package.json

```diff
@@ -142,6 +142,7 @@
     "vscode-builtin-xml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/xml-1.39.1-prel.vsix",
     "vscode-builtin-yaml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/yaml-1.39.1-prel.vsix",
     "vscode-editorconfig": "https://github.com/theia-ide/editorconfig-vscode/releases/download/v0.14.4/EditorConfig-0.14.4.vsix",
-    "vscode-eslint": "https://github.com/theia-ide/vscode-eslint/releases/download/release%2F2.0.15/vscode-eslint-2.0.15.vsix"
+    "vscode-eslint": "https://github.com/theia-ide/vscode-eslint/releases/download/release%2F2.0.15/vscode-eslint-2.0.15.vsix",
+    "vscode-references-view": "https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-references-view/references-view-0.0.47-05339d.vsix"
```

Comment:
please use this link https://github.com/theia-ide/vscode-references-view/releases/download/v0.0.47/references-view-0.0.47.vsix

For now we fork all extension under thiea-ide org, later we will consume from our registry. That's how it should be done, not pulled from somewhere else.

Reply:
Done. Sorry, I didn't spot the change in URL in your comment to #7153